scripts/change_headers.swift

```diff
@@ -20,13 +20,13 @@
 import Foundation
```

Comment:
This is a short-lived script not intended to stay in the repo long - but checked in to associate with the repo-relative changes in the PR.